pkg/models/active_node.go

```diff
@@ -5,10 +5,19 @@ type ActiveNode struct {
 	Id           int64  `json:"id"`
 	NodeId       string `json:"node_id"`
 	Heartbeat    int64  `json:"heartbeat"`
-	PartitionNo  int32  `json:"partitionNo"`
-	AlertRunType string `json:"alertRunType"`
+	PartitionNo  int32  `json:"partition_no"`
+	AlertRunType string `json:"alert_run_type"`
+	AlertStatus  string `json:"alert_status"`
 }
 
+const (
```

Comment:
these are the changes I added when I pushed my commits. Why are they showing in this diff?